_events/2024-0806-2-16-release-meetings.markdown

```diff
@@ -0,0 +1,60 @@
+---
+calendar_date: '2024-08-02'
+eventdate: 2024-08-02 09:00:00 -0700
```

Comment:
Also note you are using 9:00 here but on line 24 it says 8AM.

Reply:
Updated, I will raise a new PR as we go with the release and if anything changes with the meeting schedule.